### DISCUSSION TITLE
Get height of composed widgets

### DISCUSSION THREAD

**Discussion by PetiteViking** [ocornut/imgui#5404] [New Users Build/Link/Run issues ONLY!]
Hi,
I render different widgets based on the input data type. For data types with several components this looks something like this:
```
ImGui::Text("Component 1: ");
ImGui::SameLine();
ImGui::PushID(1);
ImGui::InputFloat(hiddenLabel, comp1Val, 0.01f, 1.0f);
ImGui::PopID();

ImGui::Text("Component 2: ");
ImGui::SameLine();
ImGui::PushID(2);
dirty |= ImGui::InputFloat(hiddenLabel, comp2Val, 0.01f, 1.0f);
ImGui::PopID();

ImGui::Text("Component 3: ");
ImGui::SameLine();
ImGui::PushID(3);
dirty |= ImGui::InputFloat(hiddenLabel, comp3Val, 0.01f, 1.0f);
ImGui::PopID();
```

Now I need to determine the height of this set of widgets. I tried surrounding them with `ImGui::BeginGroup()`/`ImGui::EndGroup()` and using `ImGui::GetItemsLineHeightWithSpacing()` to no avail. It still returns the height of a single widget. Is there a way to do this via `ImGui`?

**Comment by rokups** [comment]
`GetItemsLineHeightWithSpacing()` returns height of one line. You are looking for `GetItemRectMin()`+`GetItemRectMax()` or `GetItemRectSize()`.
